src/tools/pipelines.ts

```diff
@@ -9,23 +9,23 @@ import { BuildQueryOrder, DefinitionQueryOrder } from "azure-devops-node-api/int
 import { z } from "zod";
 import { StageUpdateType } from "azure-devops-node-api/interfaces/BuildInterfaces.js";
 
-const BUILD_TOOLS = {
-  get_builds: "build_get_builds",
-  get_changes: "build_get_changes",
-  get_definitions: "build_get_definitions",
-  get_definition_revisions: "build_get_definition_revisions",
-  get_log: "build_get_log",
-  get_log_by_id: "build_get_log_by_id",
-  get_status: "build_get_status",
+const PIPELINE_TOOLS = {
+  build_get_builds: "build_get_builds",
```

Comment:
```suggestion
  pipelines_get_builds: "build_get_builds",
```
would that makes more sense? 
- It would help on maintaining: [domain]_tool
- Our UX, builds and releases are part of pipelines, thus `pipelines_` domain makes sense prefixing all of them. 
<img width="359" height="494" alt="image" src="https://github.com/user-attachments/assets/26bc3b7c-bfc2-4ed8-9599-f32098bc50cd" />

We can always move the `build` to the middle: 
Example: 
* `build_get_log` -> `pipelines_get_build_log`

Same for rest.